adafruit_ds18x20.py

```diff
@@ -29,14 +29,15 @@
     from typing_extensions import Literal
     from circuitpython_typing import WriteableBuffer
     from adafruit_onewire.bus import OneWireBus  # pylint: disable=ungrouped-imports
+    from adafruit_onewire.device import OneWireAddress
 except ImportError:
     pass
 
 _CONVERT = b"\x44"
 _RD_SCRATCH = b"\xBE"
 _WR_SCRATCH = b"\x4E"
 _CONVERSION_TIMEOUT = const(1)
-RESOLUTION = (9, 10, 11, 12)
+RESOLUTION: tuple[Literal[9, 10, 11, 12], ...] = (9, 10, 11, 12)
```

Comment:
Declares RESOLUTION as a tuple where each value is in the set (9, 10, 11, 12).  This allows IDEs and type checkers to more strictly validate the values being accessed inside this tuple variable.